core/tx_pool.go

```diff
@@ -264,6 +278,10 @@ func NewTxPool(config TxPoolConfig, chainconfig *params.ChainConfig, chain block
 	pool.priced = newTxPricedList(pool.all)
 	pool.reset(nil, chain.CurrentBlock().Header())
```

Comment:
Do we still need this? Since we already send an initial `reset` request in the loop.

Reply:
Oh, I missed this. Will remove the call from the loop.